packages/rempl-apollo-devtools/src/subscriber/apollo-operations-tracker/apollo-operations-tracker-utils.ts

```diff
@@ -0,0 +1,28 @@
+import { DocumentNode, OperationDefinitionNode } from "graphql";
+
+export const getOperationName = (query: DocumentNode) => {
+  const definition =
+    query && query.definitions && query.definitions.length > 0
+      ? (query.definitions[0] as OperationDefinitionNode)
+      : null;
+  const operationName = definition ? definition.name?.value : "name_not_found";
+
+  return operationName;
+};
+
+export const isNumber = (input: string | number | undefined = "NA") => {
```

Comment:
Tip: make this into a TS type-guard next time, so you get benefits of type narrowing at the call-site.